### PR TITLE
improvements to commonvoice data script

### PR DESCRIPTION
**Pull request by bmwshop**
Signed-off-by: Dima Rekesh <bmwshop@gmail.com>

# What does this PR do ?
improvements to et_commonvoice_data.py: logging, handling of overwrites, tgz cleanup 

**Collection**: ASR
# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage

```python
# python get_commonvoice_data.py --num_workers=4 --language es --log --files_to_process test.tsv dev.tsv
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: scripts/dataset_processing/get_commonvoice_data.py [hunk 1]
```diff
@@ -46,13 +46,15 @@
 import sox
 from sox import Transformer
 from tqdm import tqdm
+from os.path import exists
```
Comment (titu1994) [inline scripts/dataset_processing/get_commonvoice_data.py:49]: Imprkt is and just use os.lath.exists()

**Review by titu1994 — APPROVED** [review]
Lgtm, what's up with the tests ?

File: scripts/dataset_processing/get_commonvoice_data.py (hunk 1 above)
Comment (bmwshop) [reply scripts/dataset_processing/get_commonvoice_data.py]: lol.. done

**Review by titu1994 — APPROVED** [review]
Lgtm

**Pull request merged**